src/i_main.c

```diff
@@ -40,7 +42,11 @@ int main(int argc, char **argv)
     // save arguments
 
     myargc = argc;
-    myargv = argv;
+    myargv = malloc(argc * sizeof(char *));
+
+    for (int i = 0; i < argc; i++) {
```

Comment:
Please put the brace on the next line as per coding style guidelines.

Reply:
Oops. Fixed.